src/plugin/file/kml/kmlnodelayerui.js

```diff
@@ -599,17 +782,18 @@ plugin.file.kml.KMLNodeLayerUICtrl.prototype.onCenterShapeChange = function(even
 
 
 /**
- * Handles changes to color
+ * Handle changes to opacity while it changes via slide controls
  *
- * @param {angular.Scope.Event} event
- * @param {number} value
+ * @param {?angular.Scope.Event} event
+ * @param {?} value
```

Comment:
? is very rarely a good type declaration. These should probably be numbers.

Reply:
Jeremy is correct, the all/any types should be avoided if at all possible. We also generally use the all (`*`) type vs the any (`?`) type, because the any type is completely ignored by the compiler.

https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#type-expressions

Reply:
This has been updated to properly use Number types.